src/utils/media-devices-manager.js

```diff
@@ -191,6 +200,39 @@ export default class MediaDevicesManager {
     }
   }
 
+  /**
+   * Webkit issue on iOS>13: https://bugs.webkit.org/show_bug.cgi?id=213853
+   */
+  _iOSTrackMutedHack() {
+    const iOSVersion = parseInt(browser.version.split(".")[0]);
+    if (
+      ["iOS", "Mac OS"].includes(detectedOS) &&
+      ["safari", "ios"].includes(browser.name) &&
+      iOSVersion >= IOS_MUTED_HACK_MIN_VERSION
+    ) {
+      this.audioTrack.onmute = this._muteStateChanged;
+      this.audioTrack.onunmute = this._muteStateChanged;
+    }
+  }
+
+  _muteStateChanged() {
+    if (this._audioTrack && this._audioTrack.muted) {
+      if (!this.reconnectIfUnmuted) {
+        this.wasMicEnabledBeforeMute = NAF.connection.adapter.micEnabled;
+        NAF.connection.adapter.enableMicrophone(false);
```

Comment:
FYI `NAF.connection.adapter.enableMicrophone(false)` indeed triggers the UI update when I run it in the console. But in the case of receiving a Skype call, it doesn't update the UI. I think React doesn't update the UI when the browser is not in foreground (when you have the Skype notification).